### PR TITLE
Add Java 11 support for SAXParserTest to fix #274

### LINKED ISSUE #274: Java 11 support
There is a branch `java-10-gradle` that we use for work to support Java 11 (that started with Java 10 support).

To reproduce the current state, try to install OpenJDK version 11 and compile the code from that branch, and run the tests with `./gradlew check`.

Currently 16 unit tests are still failing there. Fixes are very welcome. Some of these problems below are already referenced in an issue. If someone starts to work on a particular test, please create a separate issue referring to one or two of these failing tests:

```
    CountDownLatchTest. testCountDown
    ExecutorServiceTest. testShutdown
    SemaphoreTest. testResourceAcquisition
    ObjectStreamTest. testSimpleReadbackOk
    RuntimeTest. testAvailableProcessors
    StackWalkerTest. testCallerClass
    URLClassLoaderTest. testThrownException
    DateFormatTest. testConversionCycle
    DateFormatTest. testFormatWithTimeZone
    DateFormatTest. testSetAndGetTimeZone
    SimpleDateFormatTest. testFormatWithTimeZone
    RecursiveClinitTest. testNewInstance
    ProxyTest. testAnnoProxy
    ProxyTest. testBasicProxy
    InterruptTest. testPark
    SAXParserTest. testSimpleParse 
```

### PR DESCRIPTION
**Pull request by varad64**

### CODE REVIEW

File: src/classes/modules/java.xml/jdk/xml/internal/SecuritySupport.java [hunk 1]
```diff
@@ -0,0 +1,201 @@
+package jdk.xml.internal;
```
Comment (cyrille-artho) [inline src/classes/modules/java.xml/jdk/xml/internal/SecuritySupport.java:1]: Please add the copyright/license header here.

File: build.gradle [hunk 2]
```diff
@@ -42,12 +42,13 @@ task compileModules(type: JavaCompile) {
     dependsOn compileTestJava
     source = fileTree(dir: 'src/classes/modules')
     classpath = files('build/annotations','build/classes','build/main')
-    options.compilerArgs = [  
-        '--module-source-path', file('src/classes/modules'),
```
Comment (cyrille-artho) [inline build.gradle:46]: Please try to revert all the changes that only modify whitespace, as it makes it harder to analyze the patch in the future (or to merge it with other branches). The change should be one line (the last line).

**Review by cyrille-artho — CHANGES_REQUESTED** [review]
It's great you found a way to make this work! Please check the whitespace changes and missing license header; see my comments.

File: src/classes/modules/java.xml/jdk/xml/internal/SecuritySupport.java (hunk 1 above)
Comment (varad64) [reply src/classes/modules/java.xml/jdk/xml/internal/SecuritySupport.java:26]: Done Cyrille. Thanks for your patience!

**Comment by cyrille-artho** [comment]
Great, this passes the unit tests (on some variants of the JDK) for the first time for Java 11!
I will merge it and try this on my computer.

**Pull request merged**